### ISSUE TITLE
train and validation metrics

### ISSUE THREAD

**Issue by erwallace** [Exscientia/physicsml#17]
Hello, where/how can I specify multiple metrics I want to track during training for the train and validation datasets? And if I create custom metrics and specify them as plugins can I use any of those as well.
Cheers

**Comment by wardhaddadin1** [comment]
Hello!

Currently, the only things tracked are losses and gradients. 

It would be good to have an explicit example to understand a bit more the utility vs complexity of making something like metric computation configurable. Do you have an example? 

Thanks, 
Ward

**Comment by erwallace** [comment]
Hi Ward,
I'd like to be able to track the MAE and RMSD of the energies and forces independently for each epoch using the validation set when training a MACE model. Adding metrics for the train dataset was just out of curiosity - losses and gradients will suffice there. 
Thanks,
Ewan

**Comment by wardhaddadin1** [comment]
Ah I see! That's doable. We can easily make the lighting log each of the losses and the total loss separately (I meant to do that, but forgot actually). 

Tracking multiple metrics (not losses) per quantity is a bit trickier (will need some thought), but is a fair ask. 

Would it unblock you for now if we track the losses for each quantity alongside the total loss for now? (and try to think more about how to do general metric tracking for a future PR?)

**Comment by erwallace** [comment]
Hi Ward, 

Separate losses is a good start and would be appreciated. I’m also very happy to work with you on a PR for other metrics to speed up the process.

Thanks
